docs/src/main/asciidoc/security-authentication-mechanisms.adoc

```diff
@@ -315,7 +318,9 @@ Quarkus Security also supports the following authentication mechanisms through e
 * <<webauthn-authentication>>
 * <<openid-connect-authentication>>
 * <<smallrye-jwt-authentication>>
+ifndef::no-quarkus-elytron-security-oauth2[]
 * <<oauth2-authentication>>
+endif::no-quarkus-elytron-security-oauth2[]
 
 [[webauthn-authentication]]
```

Comment:
@rolfedh webauthn extension is not supported AFAIK

Reply:
Added `ifndef::no-webauthn-authentication[]` to hide various instances of WebAuthn in the product docs.